debugging/netcoreapp/debuggerClient.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { PlatformType } from "./osProvider";
+import { VsDbgClient } from "./vsdbgClient";
+
+export interface DebuggerClient {
+    getDebugger(os: PlatformType): Promise<string>;
+}
+
+export class DefaultDebuggerClient {
+    private static debuggerVersion: string = 'vs2017u5';
```

Comment:
Do we need settings to override these?

Reply:
To date, we haven't seen the need to make these configurable in VS.